docs/sql-ref-literals.md

```diff
@@ -0,0 +1,506 @@
+
+#### Syntax
+
+{% highlight sql %}
+DATE { 'yyyy' |
+       'yyyy-[m]m' |
+       'yyyy-[m]m-[d]d' |
+       'yyyy-[m]m-[d]d ' |
+       'yyyy-[m]m-[d]d[T]c[...]' }
+{% endhighlight %}
+
+#### Parameters
+
+<dl>
+  <dt><code><em>c</em></code></dt>
+  <dd>
+    One character from the character set.
+  </dd>
+</dl>
+
+#### Examples
+
+{% highlight sql %}
+SELECT DATE '1997' AS col;
+  +----------+
+  |       col|
+  +----------+
+  |1997-01-01|
+  +----------+
+
+SELECT TIMESTAMP '1997-01' AS col;
+  +----------+
+  |       col|
+  +----------+
+  |1997-01-01|
+  +----------+
+
+SELECT DATE '2011-11-11' AS col;
+  +----------+
+  |       col|
+  +----------+
+  |2011-11-11|
+  +----------+
+{% endhighlight %}
+
+#### Timestamp Literal
+
+#### Syntax
+
+{% highlight sql %}
+TIMESTAMP { 'yyyy' |
+            'yyyy-[m]m' |
+            'yyyy-[m]m-[d]d' |
+            'yyyy-[m]m-[d]d ' |
+            'yyyy-[m]m-[d]d[T][h]h[:]' |
+            'yyyy-[m]m-[d]d[T][h]h:[m]m[:]' |
+            'yyyy-[m]m-[d]d[T][h]h:[m]m:[s]s[.]' |
+            'yyyy-[m]m-[d]d[T][h]h:[m]m:[s]s.[ms][ms][ms][us][us][us][zone_id]' |
+            '[T][h]h:' |
+            '[T][h]h:[m]m[:]' |
+            '[T][h]h:[m]m:[s]s[.]' |
+            'T[h]h' |
+            '[T][h]h:[m]m:[s]s.[ms][ms][ms][us][us][us][zone_id]' }
+{% endhighlight %}
+`zone_id` should have one of the forms:
+<ul>
+  <li>Z - Zulu time zone UTC+0</li>
+  <li>+|-[h]h:[m]m</li>
+  <li>A short id, see <a href="https://docs.oracle.com/javase/8/docs/api/java/time/ZoneId.html#SHORT_IDS">SHORT_IDS</a></li>
+  <li>An id with one of the prefixes UTC+, UTC-, GMT+, GMT-, UT+ or UT-, and a suffix in the formats:
+    <ul>
+      <li>+|-h[h]</li>
+      <li>+|-hh[:]mm</li>
+      <li>+|-hh:mm:ss</li>
+      <li>+|-hhmmss</li>
+    </ul>
+  </li>
+  <li>Region-based zone IDs in the form <code>area/city</code>, such as <code>Europe/Paris</code></li>
+</ul>
+
+#### Examples
+
+{% highlight sql %}
+SELECT TIMESTAMP '1997-01-31 09:26:56.123' AS col;
+  +-----------------------+
+  |                    col|
+  +-----------------------+
+  |1997-01-31 09:26:56.123|
+  +-----------------------+
+
+SELECT TIMESTAMP '1997-01-31 09:26:56.66666666CST' AS col;
+  +--------------------------+
+  |                      col |
+  +--------------------------+
+  |1997-01-31 07:26:56.666666|
+  +--------------------------+
+
+SELECT TIMESTAMP '1997-01' AS col;
+  +-------------------+
+  |                col|
+  +-------------------+
+  |1997-01-01 00:00:00|
+  +-------------------+
+
+SELECT TIMESTAMP '09:26' AS col;
+  +-------------------+
+  |                col|
+  +-------------------+
+  |2020-04-17 09:26:00|
+  +-------------------+
+{% endhighlight %}
+
+### Interval Literal
+
+An interval literal is used to specify a fixed period of time.
+
+#### Syntax
+{% highlight sql %}
+{ INTERVAL interval_value interval_unit [ interval_value interval_unit ... ] |
+  INTERVAL interval_value interval_unit TO interval_unit }
```

Comment:
`interval_string_format interval_unit TO interval_unit`

Reply:
I'd call `interval_value`  here as `SQL standard year-month/date-time string` and call `interval_unit TO interval_unit` as `INTERVAL QUALIFIERS`

Reply:
I prefer to use ```interval_value```, ```interval_string_value``` and ```interval_unit``` because these are parameters and I have explanation for these parameters